apps/modernization-ui/src/routes/AppRoutes.tsx

```diff
@@ -64,6 +65,7 @@ export const AppRoutes = () => {
                 <Route path=":pageId">
                     <Route index element={<PreviewPage />} />
                     <Route path="edit" element={<Edit />} />
+                    <Route path="question-library/:subsectionId" element={<QuestionLibrary />} />
```

Comment:
As mentioned above, I don't believe the `QuestionLibrary` appears outside of a modal within the context of the `Edit` page flow. I could be wrong.

Reply:
It will be with in the context. but it is not opening in a model

Reply:
it should open in a model. So working on it

Reply:
Fixed to open in the model